src/components/app/details/triggerView/TriggerView.tsx

```diff
@@ -996,7 +996,7 @@ class TriggerView extends Component<TriggerViewProps, TriggerViewState> {
                                 stageType={DeploymentNodeType[this.state.nodeType]}
                                 material={material}
                                 materialType={this.state.materialType}
-                                envName={node.environmentName}
+                                envName={node?.environmentName}
```

Comment:
have you tested for envName as undefined

Reply:
Checked the envName to be undefined doesn't cause any error.